cmd/run.go

```diff
@@ -26,6 +27,7 @@ func newRunCommand() *cobra.Command {
 	cmd.Flags().StringVarP(&envFile, "env", "e", ".env", "Location of file containing environment variables")
 	cmd.Flags().BoolVarP(&noCache, "no-cache", "", false, "Do not use cache when building container image")
 	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "Settings file from which to import airflow objects")
+	cmd.Flags().StringVarP(&dagFile, "dag-file", "d", "", "DAG file where your DAG is located. Use this flag to parse ")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&dagFile, "dag-file", "d", "", "DAG file location which needs to be parsed to run the DAG")
```

Reply:
nit: or something else, the correct statement feels incomplete to me

Reply:
my bad